src/lib/protocols/presentproof/messages/ProofRequest.ts

```diff
@@ -0,0 +1,44 @@
+import { Expose } from "class-transformer/decorators";
+
+export class ProofRequest {
+
+    public constructor(proofRequest: ProofRequestInterface) {
+        if (proofRequest) {
+            this.name = proofRequest.name;
+            this.version = proofRequest.version;
+            this.nonce = proofRequest.nonce;
+            this.requestedAttributes = proofRequest.requestedAttributes;
+            this.requestedPredicates = proofRequest.requestedPredicates;
+        }
+    }
```

Comment:
Please check the formatting of all files with `yarn check-format` eventually fix formatting with `yarn format`.